### PR TITLE
add Norwegian grades

### LINKED ISSUE #102: add Norwegian grades
https://en.wikipedia.org/wiki/Grade_(climbing)#Scandinavian

### PR DESCRIPTION
**Pull request by josh610**
[Issue #102](https://github.com/OpenBeta/sandbag/issues/102)

Grade conversions were done via [this](https://www.thecrag.com/en/article/grades) converter from TheCrag.com

### CODE REVIEW

**Comment by musoke** [comment]
Thanks!  I'll take a look.

File: src/GradeScale.ts [hunk 1]
```diff
@@ -22,7 +22,8 @@ export const GradeScales = {
   FRENCH: 'french',
   UIAA: 'uiaa',
   EWBANK: 'ewbank',
-  SAXON: 'saxon'
+  SAXON: 'saxon',
+  NORWAY: 'norwegian'
```
Comment (musoke) [inline src/GradeScale.ts]: Let's use a consistent name everywhere.

**Comment by musoke** [comment]
Let's settle on one name used everywhere.  Probably one of
- Norwegian
- Norway
- Scandanavian

It's my understanding that Norwegian and Swedish grades are in principle different, so I would discount the last.  "Norwegian" rather than "Norway" seems more consistent with colloquial use, and with "French" instead of "France" and with "British" vs "Britain".  What do you think?

File: src/index.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ import { YosemiteDecimal, French, UIAA, Font, VScale, Ewbank } from './scales'
 // Nordic - NOR
 // Brazilian
 // Kurtyka (Poland)
+// Norwegian (aka Scandinavian)
```
Comment (musoke) [inline src/index.ts]: Could you remove this comment and the one for NOR? This is a list of grades we don't have yet.

**Review by musoke — CHANGES_REQUESTED** [review]
This looks good!  I left a couple small comments to address.

**Comment by musoke** [comment]
Was there anything about our processes that could be improved?  Is there anything we could add to docs to make writing this smoother for you?

File: src/GradeScale.ts (hunk 1 above)
Comment (josh610) [reply src/GradeScale.ts]: Settled on "Norwegian"

**Comment by josh610** [comment]
> 

To be honest, I'm not super familiar with a lot of the software and processes that Sandbag / OpenBeta uses, so I don't think I can offer constructive criticism. That being said, it seemed pretty straightforward to me.

**Review by musoke — APPROVED** [review]

**Comment by musoke** [comment]
Thank you @josh610!

@all-contributors add @josh610  for code

**Pull request merged**